### DISCUSSION TITLE
Why  farm summary  log is diffent with other people .

### DISCUSSION THREAD

**Discussion by ben00007** [Chia-Network/chia-blockchain#5999] [Support]
Why  farm log is diffent with other people 

look other people   chia farm summary log  like this .
Farming status: Farming
Total chia farmed: 12.0001
User transaction fees: 0.0001
Block rewards: 12.0
Last height farmed: 293185
Harvester count: 5
Plot count: 13145
Total size of plots: 1.271 PiB
Estimated network space: 9025.210 PiB
Expected time to win: 1 day and 8 hours
Note: log into your key using 'chia wallet show' to see rewards for each key

my env 
ubuntu 20.04
chia 1.1.6

but my log is this .

Farming status: Farming
Total chia farmed: 20
User transaction fees: 0.015459900032
Block rewards: 20
Last height farmed: 332484
Plot count: 159
Total size of plots: 15.738 TiB
Estimated network space: 10639.252 PiB
Expected time to win: 4 months and 3 weeks

my  summary log  haven't  " Harvester count "

**Comment by BrandtH22** [comment]
@ben00007 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

Note the harvester count indicates how many total harvesters you have, if you are farming on just one machine it is expected that might not be present.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
